02_classes-objects-methods/02_03_car.py

```diff
@@ -6,3 +6,27 @@
 #
 # Create at least two different objects of this `Car()` class and demonstrate
 # changing the objects' attributes.
+
+class Car:
+
+    def __init__(self, model, year, max_speed) -> None:
+        self.model = model
+        self.year = year
+        self.max_speed = max_speed
+
+    def newspeed(self):
+        self.max_speed += 5
+        return self.max_speed
```

Comment:
You don't really need to return max_speed here. because you always have access to it through the `.max_speed` attribute of the class instance.
I don't see any reason to return this.